zxlive/edit_panel.py

```diff
@@ -60,6 +61,18 @@ def _toolbar_sections(self) -> Iterator[ToolbarSection]:
         self.start_derivation.setText("Start Derivation")
         self.start_derivation.clicked.connect(self._start_derivation)
         yield ToolbarSection(self.start_derivation)
+        
+        self.dodo_hint = QToolButton(self)
+        self.dodo_hint.setIcon(QIcon(get_data("icons/dodo.png")))
+        self.dodo_hint.setToolTip("Dodo Hint")
```

Comment:
Although this is funny, maybe make it more descriptive? "Ask DodoGPT for suggestions on how to rewrite your diagram"

Reply:
Sure!